packages/vuetify/src/scss/variables/_colors.scss

```diff
@@ -0,0 +1,320 @@
+$red: (
+  'base': #F44336,
+  'lighten-5': #FFEBEE,
```

Comment:
Maybe use scss `lighten`, `darken` and `saturate` functions to compute these colors?

Reply:
https://material.io/design/color/the-color-system.html#color-usage-palettes

I don't know if there's a particular pattern behind them, even material-components-web hardcodes them. It doesn't matter anyway as they aren't intended to be changed.

Reply:
@KaelWD okay